sunpy/map/sources/tests/test_mdi_source.py

```diff
@@ -23,6 +26,94 @@ def mdi():
     return Map(fitspath)
 
 
+@pytest.fixture
+def mdi_synoptic():
+    # sample MDI map
+    # Please do not edit this header, it is a 1:1 copy of a MDI map header
+    header = dedent("""
+        # HDU 0 in mdi.synoptic_mr_small_96m.2099.data.fits:
```

Comment:
Would be better to keep this in a separate txt file?

Reply:
I think others are just in the test code files? Happy to move it to a text file, but not sure where the best place to put that would be.

Reply:
I'm happy to keep it here over a text file.